addons/mail_bot/data/mailbot_data.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data>
-        <record id="base.user_root" model="res.users">
-            <field name="odoobot_state">disabled</field>
```

Comment:
I think this one should be kept, we don't want odoobot to start the tour with itself (when logged as system).